pkg/checks/daemonSet/daemonSet_test.go

```diff
@@ -36,6 +36,27 @@ func TestCleanupOrphans(t *testing.T) {
 	}
 }
 
+func TestPauseContainerOverride(t *testing.T) {
+	// verify that we are getting the expected default value from a new dsc
+	dsc, err := New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if dsc.PauseContainerImage != "gcr.io/google_containers/pause:0.8.0" {
+		t.Fatal("Default Pause Container Image is not set or an unexpected value, actual value:", dsc.PauseContainerImage)
+	}
+
+	dscO, err := New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Silly, yes, but this mimics how the program is setting the override value
```

Comment:
not sillly!  We are testing the current functionality.  The future may break any unknown functionality that may change behavior.